third_party/onnx.BUILD

```diff
@@ -52,7 +52,6 @@ cc_library(
             "onnx/defs/tensor/*.cc",
             "onnx/defs/traditionalml/*.cc",
             "onnx/defs/training/defs.cc",
-            "onnx/optimizer/*.cc",
```

Comment:
Does this have to be removed here? This will be updated once we update the ONNX submodule.

Reply:
Good catch. Thank you!